client/blocks/app-banner/style.scss

```diff
@@ -0,0 +1,68 @@
+.card.app-banner {
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $blue-light;
+	color: $blue-dark;
+	/* Needed for banner to appear in front of notifications panel. */
+	z-index: z-index( 'root', '.app-banner' );
+
+	background-repeat: no-repeat;
+	/*
+	 * Needed to match the design as closely as possible,
+	 * we want the background image to be smaller than the containing div.
+	 */
+	background-size: 270px;
+	background-position: right 0 bottom 0;
+}
+
+.card.app-banner.post-editor {
+	background-image: url( '/calypso/images/illustrations/app-banner-editor.svg' );
+}
+
+.card.app-banner.notifications {
+	background-image: url( '/calypso/images/illustrations/app-banner-notifications.svg' );
+}
+
+.card.app-banner.reader {
+	background-image: url( '/calypso/images/illustrations/app-banner-reader.svg' );
+}
+
+.card.app-banner.stats {
+	background-image: url( '/calypso/images/illustrations/app-banner-stats.svg' );
+}
+
+.app-banner__text-content {
+	width: calc( 100% - 84px );
+	min-height: 66px;
+}
+
+.app-banner__title {
+	font-size: 17px;
+	font-weight: 600;
+	line-height: 1.4;
+}
+
+.app-banner__copy {
+	font-size: 12px;
+	font-weight: 400;
+}
+
+.app-banner__buttons {
+	width: calc( 100% - 84px );
+}
+
+.button.app-banner__open-button {
+	margin-top: 8px;
+}
+
+a.app-banner__no-thanks-button {
+	color: $blue-dark;
+	display: inline-block;
+	margin-top: 8px;
+	padding: 7px 14px 9px;
```

Comment:
I tested with iPhone 5 size viewport and saw the wrapping "No thanks" button.

<img width="350" alt="screen shot 2017-07-19 at 10 09 03" src="https://user-images.githubusercontent.com/908665/28360047-354ee854-6c6c-11e7-91c9-091fd4d7a77f.png">

We could reduce this to happen by removing one side of padding from the button.

<img width="340" alt="screen shot 2017-07-19 at 10 20 37" src="https://user-images.githubusercontent.com/908665/28360054-3fc346fe-6c6c-11e7-9700-133a13064735.png">